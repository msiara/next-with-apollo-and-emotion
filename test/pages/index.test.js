import { shallow } from 'enzyme';
import React from 'react';
import Index from '../../src/pages';

describe('<Index />', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Index />);
    expect(wrapper).toMatchSnapshot();
  });
});
