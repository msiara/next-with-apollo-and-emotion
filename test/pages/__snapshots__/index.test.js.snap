// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> should match snapshot 1`] = `"Next with Apollo and Emotion"`;
